src/ui/hexedit.cc

```diff
@@ -879,12 +1059,20 @@ bool HexEdit::isByteVisible(qint64 bytePos) {
   return isRangeVisible(bytePos, 1);
 }
 
-void HexEdit::scrollToByte(qint64 bytePos, bool doNothingIfVisable) {
-  if (isByteVisible(bytePos) && doNothingIfVisable) {
+void HexEdit::scrollToByte(qint64 bytePos, bool minimal_change) {
```

Comment:
Slight note about scrolling behaviour - it is only done after whole byte is edited, so one can edit something without seeing it (and maybe not being aware of it he edited it by mistake).